Libraries/Text/TextInput/Singleline/RCTUITextField.m

```diff
@@ -176,6 +176,16 @@ - (CGRect)editingRectForBounds:(CGRect)bounds
 
 #pragma mark - Overrides
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-implementations"
+// Overrides selectedTextRange setter to get notify when selectedTextRange changed
+- (void)setSelectedTextRange:(UITextRange *)selectedTextRange
+{
+  [super setSelectedTextRange:selectedTextRange];
+  [_textInputDelegateAdapter selectedTextRangeWasSet];
```

Comment:
Should we try to call delegates in the same order? In the method below this one, we call the delegate method and then we call super; here, it's reversed.

Reply:
🤔 Emm, may not have much difference IMO, but I can consolidate them.

Reply:
@JoshuaGross I reverted to this change, we need to call super first, otherwise, `onSelectionChange` would be called with last args, not latest args.